enzyme/benchmarks/ReverseMode/ba/Makefile.make

```diff
@@ -1,23 +1,17 @@
-# RUN: cd %S && LD_LIBRARY_PATH="%bldpath:$LD_LIBRARY_PATH" BENCH="%bench" BENCHLINK="%blink" LOAD="%loadEnzyme" make -B ba-unopt.ll ba-raw.ll results.json -f %s
+# RUN: cd %S && LD_LIBRARY_PATH="%bldpath:$LD_LIBRARY_PATH" BENCH="%bench" BENCHLINK="%blink" LOAD="%newLoadClangEnzyme" make -B ba.o results.json -f %s
 
 .PHONY: clean
 
+dir := $(abspath $(lastword $(MAKEFILE_LIST))/../../../..)
+
 clean:
 	rm -f *.ll *.o results.txt results.json
 
-%-unopt.ll: %.cpp
-	clang++ $(BENCH) $^ -O2 -fno-vectorize -fno-slp-vectorize -ffast-math -fno-unroll-loops -Xclang -new-struct-path-tbaa -o $@ -S -emit-llvm
-	#clang++ $(BENCH) $^ -O1 -Xclang -disable-llvm-passes -fno-vectorize -fno-slp-vectorize -ffast-math -fno-unroll-loops -Xclang -new-struct-path-tbaa -o $@ -S -emit-llvm
-
-%-raw.ll: %-unopt.ll
-	opt $^ $(LOAD) -enzyme -o $@ -S
-
-%-opt.ll: %-raw.ll
-	opt $^ -o $@ -S
-	#opt $^ -O2 -o $@ -S
+$(dir)/benchmarks/ReverseMode/ba/target/release/libbars.a: src/lib.rs Cargo.toml
```

Comment:
if this can be optional?

Reply:
What would you like this variable to use when determining if Rust is enabled? We could run `cargo +enzyme version` (checking whether a toolchain named `enzyme` is configured) and/or we could have an environment variable to be set (by CMake or otherwise).